src/main/java/com/evolvedbinary/tulip/XPath10Lexer.java

```diff
@@ -15,9 +15,44 @@
 package com.evolvedbinary.tulip;
 
 import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
 
 public class XPath10Lexer extends AbstractLexer {
 
+    public static Trie trie = new Trie();
+
+    static {
+        // Insert all axis names
+        for (String axis : new String[]{
```

Comment:
The `axis` are just specific Token Types that are also Reserved Words. They don't need to be treated differently to any other reserved words.